tests/Eluceo/iCal/Component/CalendarIntegrationTest.php

```diff
@@ -141,4 +141,55 @@ public function testRenderIsIdempotent()
 
         $this->assertEquals($vCalendar->render(), $vCalendar->render());
     }
+    
+    /**
+     * This test was introduced because of the added IMAGE property to the Event class.
+     *
+     * @covers \Eluceo\iCal\Component\Event::addImage()
+     */
+    public function testEventAddImage()
+    {
+        $vCalendar = new \Eluceo\iCal\Component\Calendar('www.example.com');
+
+        $vEvent = new \Eluceo\iCal\Component\Event('123456');
+        $vEvent->setDtStart(new \DateTime('2012-12-24'));
+        $vEvent->setDtEnd(new \DateTime('2012-12-24'));
+        $vEvent->setSummary('Vacations');
+        $image = Array(
```

Comment:
There should probably be a new image type

Reply:
Yeah it would be easier to use with setters 👌